tools/coq_makefile.ml

```diff
@@ -201,14 +201,9 @@ let windrive s =
 
 let generate_conf_coq_config oc args bypass_API =
```

Comment:
We should remove the coq_makefile command line flag entirely.

Reply:
I disagree:
- people may use it (even if I removed it from the 3 bench that were failing)
- we still hope to have an API one day

Reply:
>we still hope to have an API one day

Do we? I thought we were going with the Public/Internal stuff.

Also following Coq master without being in Coq's CI sounds like frustrating and fruitless work.